### ISSUE TITLE
Request: Add repositories with Bible translations.

### ISSUE THREAD

**Issue by ajes** [yukuku/androidbible#75]
It would be nice to have option in the application, to enter URL for additional repository with unofficial YES files, made by community.
Something like packages repo for Linux distributions. You have official repo maintained by Distribution maintainers, and you have community repos.

The functionality should allow add as many as you want repos, by using URL or QR code.
List of repos should allow modifying it (changing URL or even priority), enabling and disabling it, and finally removing repo.

**Comment by AlijiEmmanuel-dev** [comment]
This is a very key feature.
I have been wanting this for years now

**Comment by yukuku** [comment]
I agree this is an important feature. Does anyone have the bandwith to implement this feature?

**Comment by ajes** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

you mean time to implement it, or place for storing files and bandwidth for downloading?


Wysłano z bezpiecznej poczty e-mail Proton Mail.

------- Original Message -------
środa, 31 sierpnia 2022 08:58, Randy Sugianto (Yuku) ***@***.***> napisał(a):


> I agree this is an important feature. Does anyone have the bandwith to implement this feature?
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.
-----BEGIN PGP SIGNATURE-----
Version: ProtonMail

wnUEARYKAAYFAmMPBwQAIQkQP1g+OhBz7kUWIQTSWXF4J1vgcFBNcSo/WD46
EHPuRcQhAPwMcU87bhcBKKtGw+8MNx47aLGE5aa69bBaKdqwcM301gD+LflE
hD/OW7SSJOxuoPLdIabRS7iOnRo+MGxzMAjnaws=
=1kVm
-----END PGP SIGNATURE-----

**Comment by yukuku** [comment]
Time to implement it on the app.
